src/cmr/concepts/collection.js

```diff
@@ -21,6 +21,7 @@ export default class Collection extends Concept {
    * @param {String} id Concept ID to set a value for within the result set
    * @param {Object} item The item returned from the CMR json endpoint
    */
+  // TODO: Since all concepts now support associations we should refactor this on the parent class
```

Comment:
What would be the LOE to take care of that in this ticket so we can take care of the TODO?

Reply:
Changing this would only allow us to set associations as a default component of all the concepts, we would still have to explicitly write code to allow the associations to flow through. That would be a different ticket which would be to refactor graphql so that you can add an association just by adding the type to the .graphql file thus reducing some boilerplate.

Reply:
Looking at this I will go ahead and write a ticket for it. I think we'll want to add `association_details` to all the concepts as well since they all support that field. EXCEPT granules. I am a bit concerned looking at it closer that it will cause an issue on granule searching with `graphql` I don't THINK it will, since the field will just always be null the and get caught, the same way it works for concept's which support `association_details` but, are not associated to anything and thus, don't actually have the field in the response.